CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+# 0.10.0 / 2022-10-12
```

Comment:
One small one I noticed missing is `create panel preview component #604`, not really I full feature but it could fall under enhancement

Reply:
right we talked about this, don't know how i missed it 😅

Reply:
no worries! thanks again for doing this release, hopefully `make tag` and the build works 🤞🤞